### PR TITLE
clean towr step branch && add guide to wheeled robot in readme

### PR DESCRIPTION
**Pull request by chaofiber**
Cleaned the wheel_step branch and update README to include guide to wheeled robot.

### CODE REVIEW

**Comment by mbjelonic** [comment]
@chaofiber what about the PR in xpp?

File: towr_ros/CMakeLists.txt [hunk 1]
```diff
@@ -4,148 +4,161 @@ project(towr_ros)
...
-  xpp_states
-  towr
+        roscpp
+        rosbag
+        std_msgs
+        visualization_msgs
+        message_generation
+        xpp_msgs
+        xpp_states
+        towr
```
Comment (mbjelonic) [inline towr_ros/CMakeLists.txt]: @chaofiber can you please revoke the formatting changes here?

**Comment by chaofiber** [comment]
> 
> 
> @chaofiber what about the PR in xpp?

Do you mean we should also mention that change the xpp branch in readme?

File: towr_ros/CMakeLists.txt (hunk 1 above)
Comment (mbjelonic) [reply towr_ros/CMakeLists.txt]: Similarly please do this for the whole file.

File: towr/src/nlp_formulation.cc [hunk 2]
```diff
@@ -166,149 +166,192 @@ NlpFormulation::MakeEndeffectorVariables ()
...
-      polys_per_phase_force.emplace_back(fnodes);
-      polys_per_phase_decision.emplace_back(dnodes);
-
-      params_.number_of_polys_per_phase_motion_.at(ee) = polys_per_phase_motion;
-      params_.number_of_polys_per_phase_force_.at(ee) = polys_per_phase_force;
-      params_.number_of_polys_per_phase_decision_.at(ee) = polys_per_phase_decision;
-
-      params_.ee_phase_durations_.at(ee) = durations;
-    }
+     if(final_base_v_.lin.p().y()==0 && final_base_v_.ang.p().z()==0 ) {
```
Comment (mbjelonic) [inline towr/src/nlp_formulation.cc]: @chaofiber why are here so many changes? If you did not change here anything please make sure that it doe not inidcated the whole file as being changed.

**Comment by chaofiber** [comment]
oh did I just forget to create the xpp PR?  I will check.:(

**Comment by mbjelonic** [comment]
@chaofiber yes, you declined the PR in xpp.

**Comment by chaofiber** [comment]
@mbjelonic just created xpp PR again.

File: towr_ros/CMakeLists.txt (hunk 1 above)
Comment (chaofiber) [reply towr_ros/CMakeLists.txt]: Done, updated the PR.

File: towr/src/nlp_formulation.cc (hunk 2 above)
Comment (chaofiber) [reply towr/src/nlp_formulation.cc]: Reversed the change, and fixed the type of initial code; added the stepFlat specific-initialization

**Pull request merged**